webpack/scenes/FlatpakRemotes/CreateEdit/FlatpakRemoteform.js

```diff
@@ -68,12 +70,19 @@ const FlatpakRemotesForm = ({ setModalOpen, remoteData }) => {
 
   const onSave = () => {
     setSaving(true);
-    dispatch(createFlatpakRemote({
-      name,
-      url,
-      username,
-      password,
-    }));
+    dispatch(isEditing ?
+      updateFlatpakRemote(editingId, {
+        name,
+        url,
+        username,
+        password,
+      }, () => window.location.assign('/flatpak_remotes/')) :
+      createFlatpakRemote({
+        name,
+        url,
+        username,
+        password,
```

Comment:
```suggestion
        token: password,
```
Same reason as above.